### PR TITLE
Change 'He' to 'They'

### PR DESCRIPTION
**Pull request by loolooyyyy**

### CODE REVIEW

File: docs/basics.rst [hunk 1]
```diff
@@ -16,7 +16,7 @@ subsequently be ready to accept form submission input are:
 - Render the form.
 
 Once the form is rendered, a user will interact with the form in his
-browser, and some point, he will submit it.  When the user submits the
+browser, and some point, they will submit it.  When the user submits the
```
Comment (tseaver) [inline docs/basics.rst]: Maybe better to drop the pronoun altogether here?  E.g.

    browser, and at some point, will submit it.

Note also the `his` at the end of the previous line.

File: docs/basics.rst [hunk 2]
```diff
@@ -448,7 +448,7 @@ Validating a Form Submission
 ----------------------------
 
 Once the user seen the form and has chewed on its inputs a bit, he
-will eventually submit the form.  When he submits it, the logic you
+will eventually submit the form.  When they submit it, the logic you
```
Comment (tseaver) [inline docs/basics.rst]: I'd rather rewrite to remove the redundancy, e.g.:

    will eventually submit the form.  At that point, the logic you

(Again, note the `he` at the end of the previous line).

**Comment by tseaver** [comment]
Thanks for the suggested improvements!

**Comment by miohtama** [comment]
Thank you for your pull request.

I am the new Deform project maintainer. I apologize the issue of this pull request not being commented earlier. Deform project is again under activate maintenance.

This is issue tracker / pull request clean to prepare a new release.

This pull request is good to go.

**Comment by tseaver** [comment]
@miohtama you've merged this PR with my comments unaddressed (leaving other masculine pronouns in the same paragraphs, for instance).

**Comment by miohtama** [comment]
@tseaver Thanks!. I'll address this later in another commit with some more documentation clean up.

**Pull request merged**